ViewModel/HyvaGridInterface.php

```diff
@@ -17,6 +17,7 @@ public function getColumnDefinitions(): array;
     public function getAllColumnDefinitions(): array;
 
     /**
+     * @param bool $forceReload
```

Comment:
Please remove the `@param` annotation.

Reply:
I've changed it maybe it works better now